packages/jest-runner/corpus.ts

```diff
@@ -18,49 +18,57 @@ import path from "path";
 import fs from "fs";
 
 export class Corpus {
-	private readonly _inputDirectory: string;
-	private readonly _outputDirectory: string;
+	private readonly _inputsDirectory: string;
 
 	constructor(testPath: string, testStateElements: string[]) {
```

Comment:
I think the code can be made clearer by renaming the parameters to something like: 
```typescript
constructor(testSourceFilePath: string, testJestPathElements: string[])
```

Reply:
Done.